industrial_ci/src/tests/source_tests.sh

```diff
@@ -39,8 +39,13 @@ ROSWS=wstool
 ici_time_end  # init_ici_environment
 
 function catkin {
+  PATH_CATKIN=/usr/bin/catkin
+  PKG_CATKIN=python-catkin-tools
   local cmd=$1
   shift
+  if [ ! -f ${PATH_CATKIN} ]; then
+  	echo "[ERR] ${PATH_CATKIN} not available. Make sure ${PKG_CATKIN} is installed. See also https://github.com/ros-industrial/industrial_ci/issues/216";  	
```

Comment:
The error function should be used instead. I will take care of the colors and the exit code.

Reply:
Yeah, I just couldn't find `err` was used in the same .sh file so I tried not to use it.